pyodide_build/out_of_tree/venv.py

```diff
@@ -251,17 +288,27 @@ def install_stdlib(venv_bin: Path) -> None:
     check_result(result, "ERROR: failed to install unvendored stdlib modules")
 
 
-def create_pyodide_venv(dest: Path) -> None:
+def create_pyodide_venv(dest: Path, virtualenv_args: list[str] | None = None) -> None:
     """Create a Pyodide virtualenv and store it into dest"""
     logger.info("Creating Pyodide virtualenv at %s", dest)
     from virtualenv import session_via_cli
 
-    if dest.exists():
-        logger.error("ERROR: dest directory '%s' already exists", dest)
-        sys.exit(1)
-
     interp_path = pyodide_dist_dir() / "python"
-    session = session_via_cli(["--no-wheel", "-p", str(interp_path), str(dest)])
+
+    cli_args = ["--python", str(interp_path)]
+
```

Comment:
I changed the default CLI args to also include `wheel` (removed `--no-wheel`). I'm not sure why this was included when this code was written, as I looked at https://github.com/pyodide/pyodide-build/commit/07da853331b6066cb0c77642990a4c6b63337704 and pyodide/pyodide#2976 and couldn't find the reason. Maybe you might be able to recall this, @hoodmane?

Also, I changed `-p` to `--python` as it is more expressive.

Reply:
Well there's the question of whether we want `wheel` to run in native Python or in Pyodide's Python. Presumably it should run in the native Python? But that requires extra code to replace the shebang in `bin/wheel`. Also, it's weird magic. It's necessary for pip, but people can run wheel from a normal Python runtime, or if they really want to run it in Pyodide they have the option to `bin/pip install wheel`. So overall, I just don't see the reason for including it.

Reply:
Yeah, I think it is always better to run the `wheel` CLI in native Python. I was just interested to know if there was a specific reason not to include it in compatibility terms, so I'll drop it for now. Should we do this for `setuptools` as well using `--no-setuptools`, as I don't see a reason to include it either?

Reply:
Never mind, `wheel` and `setuptools` are not being included – I had turned them off in the Typer CLI definitions at some point when working on this PR, so we should be good.